packages/api/pages/api/languages/[code]/import.ts

```diff
@@ -32,6 +32,7 @@ export default createRoute<{ code: string }>()
       }
 
       res.ok({
+        userId: job.userId ?? undefined,
```

Comment:
We don't need to send this data to the frontend

Reply:
Done